### PR TITLE
Don't OOV_SCORE on empty prefix

### LINKED ISSUE #2579: Silence is inferred as "i"
I am getting `i` as an output for silence.

- **OS Platform**: Archlinux
- **Python version**: 3.8.0

<details><summary>Followed these steps for deepspeech installation</summary>

```bash

virtualenv -p python3 $HOME/tmp/deepspeech-venv/
source $HOME/tmp/deepspeech-venv/bin/activate

pip3 install deepspeech

curl -LO https://github.com/mozilla/DeepSpeech/releases/download/v0.6.0/deepspeech-0.6.0-models.tar.gz
tar xvf deepspeech-0.6.0-models.tar.gz
```

</details>

- **Commands to reproduce**:

First to create 400ms silence wav file:
`sox -n -r 16k -c1 -b 16 400ms-silence.wav trim 0.0 0.4`

`deepspeech --model deepspeech-0.6.0-models/output_graph.pbmm --lm deepspeech-0.6.0-models/lm.binary --trie deepspeech-0.6.0-models/trie --audio 400ms-silence.wav --beam_width 500 --lm_alpha 0.75 --lm_beta 1.85`

<details><summary>Output</summary>

```
Loading model from file deepspeech-0.6.0-models/output_graph.pbmm
TensorFlow: v1.14.0-21-ge77504a
DeepSpeech: v0.6.0-0-g6d43e21
Loaded model in 0.0107s.
Loading language model from files deepspeech-0.6.0-models/lm.binary deepspeech-0.6.0-models/trie
Loaded language model in 0.000166s.
Running inference.
i
Inference took 0.373s for 0.400s audio file.
```

</details>

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #2579

### CODE REVIEW

**Comment by lissyx** [comment]
@abdul-rehman0 tested and confirmed this was fixing for him

File: native_client/ctcdecode/ctc_beam_search_decoder.cpp [hunk 1]
```diff
@@ -170,7 +170,8 @@ DecoderState::decode() const
     for (size_t i = 0; i < beam_size_ && i < prefixes_copy.size(); ++i) {
       auto prefix = prefixes_copy[i];
       if (prefix->is_empty()) {
-        scores[prefix] = OOV_SCORE;
+        // TODO: doing 'scores[prefix] = OOV_SCORE;'
+        // generates 'i' instead of silences
```
Comment (reuben) [inline native_client/ctcdecode/ctc_beam_search_decoder.cpp]: No need to leave the TODO here, we can just outright remove the conditional.

Comment (lissyx) [reply native_client/ctcdecode/ctc_beam_search_decoder.cpp]: So this does not regresses any UTF-8 work you did, right ?

Comment (reuben) [reply native_client/ctcdecode/ctc_beam_search_decoder.cpp]: No, and even if it does I'd have to find a proper solution anyway.

**Pull request merged**